frontend/tests/controllers-test/challengeListCtrl.test.js

```diff
@@ -1,368 +1,207 @@
 'use strict';
 
-describe('Unit tests for challenge list controller', function () {
-    beforeEach(angular.mock.module('evalai'));
-
-    var $controller, createController, $rootScope, $scope, utilities, vm;
-
-    beforeEach(inject(function (_$controller_, _$rootScope_, _utilities_,) {
-        $controller = _$controller_;
-        $rootScope = _$rootScope_;
-        utilities = _utilities_;
-
-        $scope = $rootScope.$new();
-        createController = function () {
-            return $controller('ChallengeListCtrl', {$scope: $scope});
+angular.module('evalai')

```

Comment:
Hey @wahid18-maqs , Can you please explain the code you are removing in the tests? and how you are adding the current tests?